test/unit/component.js

```diff
@@ -1,189 +1,211 @@
-import ShopifyBuy from '../../src/buybutton';
 import Component from '../../src/component';
 import View from '../../src/view';
+import Updater from '../../src/updater';
 import componentDefaults from '../../src/defaults/components';
 
-describe('Component class', () => {
 
+describe('Component class', () => {
   describe('constructor', () => {
-    it('stores id, node, and handle on instance', () => {
-      let node = document.createElement('div');
-      let component = new Component({
-        id: 1234,
-        handle: 'lol',
-        node: node,
-      });
+    let component;
+    const node = document.createElement('div');
+
+    beforeEach(() => {
+      component = new Component({
```

Comment:
Let's abstract this out to a constant and use the constant reference in our assertions